share/spice/tor_node/tor_node.js

```diff
@@ -79,33 +150,47 @@
         }
 
         // Configure the template.
-        base.templates.group = "text";
-	base.templates.options.content = "record";
-	base.templates.options.keySpacing = true;
-        base.meta.sourceUrl = node.url;
+        base.templates = {
+            group: "text",
+            options: {
+                content: "record",
+                keySpacing: true
+            },
+            meta: {
```

Comment:
Can we add 
```
itemType: "Nodes"
```

![selection_298](https://cloud.githubusercontent.com/assets/5282264/6764168/fc6e81fc-cfdd-11e4-99f1-4922269d8f29.png)